tests/spdx3/bump/test_file_bump.py

```diff
@@ -24,8 +24,11 @@ def test_bump_file(creation_information):
     integrity_method: Hash = Hash(HashAlgorithm.SHA1, "71c4025dd9897b364f3ebbb42c484ff43d00791c")
 
     payload = Payload()
-    bump_file(spdx2_file, payload, creation_information=creation_information)
-    file = payload.get_element(file_fixture().spdx_id)
+    document_namespace = "https://doc.namespace"
+    bump_file(spdx2_file, payload, creation_information, document_namespace)
+    expected_new_file_id = document_namespace + "#" + file_fixture().spdx_id
```

Comment:
I would suggest to reorder this a bit to follow a given-when-then test structure. 
```
document_namespace = ..
expected_new_file_id = ...

bump_file(...)
```
Would also apply to the other tests.

Reply:
done